Makefile

```diff
@@ -25,6 +25,9 @@ CEPHFS_IMAGE_VERSION=$(if $(ENV_CEPHFS_IMAGE_VERSION),$(ENV_CEPHFS_IMAGE_VERSION
 CSI_IMAGE_NAME?=quay.io/cephcsi/cephcsi
 CSI_IMAGE_VERSION?=v1.0.0
 
+CSI_IMAGE_NAME?=quay.io/cephcsi/cephcsi
```

Comment:
@humblec looks like we have duplicate lines in Makefile

Reply:
@Madhu-1 checking

Reply:
@Madhu-1 corrected. Thanks!